xcube/core/store/stores/s3.py

```diff
@@ -59,13 +61,16 @@
     '.levels': (TYPE_SPECIFIER_MULTILEVEL_DATASET, 'levels', _STORAGE_ID),
 }
 
+_REGISTRY_FILE = 'registry.json'
```

Comment:
What is the benefit of another JSON file? In Zarr, all metadata is already in JSON format.